src/model/car.js

```diff
@@ -0,0 +1,23 @@
+import { CAR } from '../utils/constants.js';
+
+export class Car {
```

Comment:
하나의 객체만 내보낸다면 
```suggestion
export default class Car {
```
처럼 `default`를 추가하시는 것이 어떨까요?